src/test/java/picard/sam/FilterSamReadsTest.java

```diff
@@ -99,7 +99,7 @@ public void testPairedIntervalFilter(final String intervalFilename, final int ex
         sortedSamIdx.deleteOnExit();
 
         final SAMFileWriter writer = new SAMFileWriterFactory()
-                .setCreateIndex(true).makeBAMWriter(builder.getHeader(), false, inputSam);
+                .setCreateIndex(true).makeSAMWriter(builder.getHeader(), false, inputSam);
```

Comment:
This change also stops the test from leaving index files lying around. It also seems to make more sense given the `inputSam` name and extension, but I honestly don't know enough about BAM vs SAM to know if this is the right move. The test passes either way.